pkg/devspace/services/synccontroller/controller.go

```diff
@@ -295,14 +298,38 @@ func (c *controller) initClient(pod *v1.Pod, container string, syncConfig *lates
 		options.ExcludePaths = syncConfig.ExcludePaths
 	}
 
+	if syncConfig.ExcludeFile != "" {
+		paths, err := parseExcludeFile(filepath.Join(syncConfig.LocalSubPath, syncConfig.ExcludeFile))
+		if err != nil {
+			return nil, errors.Wrap(err, "parse exclude file")
+		}
+		options.ExcludePaths = append(options.ExcludePaths, paths...)
+	}
+
 	if len(syncConfig.DownloadExcludePaths) > 0 {
 		options.DownloadExcludePaths = syncConfig.DownloadExcludePaths
 	}
 
+	if syncConfig.DownloadExcludeFile != "" {
+		paths, err := parseExcludeFile(filepath.Join(syncConfig.LocalSubPath, syncConfig.DownloadExcludeFile))
+		if err != nil {
+			return nil, errors.Wrap(err, "parse download exclude file")
+		}
+		options.DownloadExcludePaths = append(options.DownloadExcludePaths, paths...)
+	}
+
 	if len(syncConfig.UploadExcludePaths) > 0 {
 		options.UploadExcludePaths = syncConfig.UploadExcludePaths
 	}
 
+	if syncConfig.UploadExcludeFile != "" {
+		paths, err := parseExcludeFile(filepath.Join(syncConfig.LocalSubPath, syncConfig.UploadExcludeFile))
+		if err != nil {
+			return nil, errors.Wrap(err, "parse upload exclude file")
+		}
+		options.UploadExcludePaths = append(options.DownloadExcludePaths, paths...)
```

Comment:
Should be options.UploadExcludePaths. Also what happens if `options.UploadExcludePaths` is nil?